tutorial-week-4.md

```diff
@@ -1,13 +1,61 @@
 
 
-Maybe today I can write something different.
+# DOM
```

Comment:
Definitely add some set up here... why are you talking about the DOM? Why is it important to understand? What exactly is this tutorial?